File: ros_modules/vlp16_ws/template/node.cpp
#include "rclcpp/rclcpp.hpp"

class CustomNode: public rclcpp::Node
{
public:
    CustomNode(): Node("node_name")
    {

    }
private:
    
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<CustomNode>();   
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: ros_modules/vlp16_ws/template/node.py
import rclpy
from rclpy.node import Node

class CustomNode(Node):
    
    def __init__(self):
        super().__init__("node_name")
        

def main(args=None):
    
    rclpy.init(args=args)
    node = CustomNode()
    rclpy.spin(node) # keeps the node alive
    rclpy.shutdown()

if __name__ == '__main__':
    main()